### PR TITLE
Add feature flags

### PR DESCRIPTION
**Pull request by fofr**
Allow us to turn off:

- nationality
- international addresses
- address lookup
- international qualifications

### CODE REVIEW

File: app/views/application/personal-details/index.njk [hunk 1]
```diff
@@ -76,28 +76,30 @@
...
+          text: "Second nationality"
+        },
+        items: nationalities
+      } | decorateApplicationAttributes(["candidate", "second-nationality"])) }}
+    {% endset -%}
+
+    {{ govukDetails({
+      summaryText: "Add another nationality",
+      html: anotherNationality,
+      open: ["candidate", "second-nationality"] | ifSetForApplication
```
Comment (paulrobertlloyd) [inline app/views/application/personal-details/index.njk]: What does `ifSetForApplication` do? I’d not spotted this before.

File: app/views/application/personal-details/index.njk [hunk 2]
```diff
@@ -76,28 +76,30 @@
...
-  {{ appAutocomplete({
-    label: {
-      text: "Nationality",
-      classes: "govuk-label--m"
-    },
-    items: nationalities
-  } | decorateApplicationAttributes(["candidate", "nationality"])) }}
-
-  {% set anotherNationality %}
+  {% if data.flags.nationality %}
```
Comment (paulrobertlloyd) [inline app/views/application/personal-details/index.njk]: We will still want to ask for nationality in the MVP, but not do any branching branch based on this answer.

**Comment by paulrobertlloyd** [comment]
Really impressed by how little needed to be changed to support these different variations!

**Comment by fofr** [comment]
> For international address, if international_address is true, show show structured fields with internationalised labels (i.e. ’Zip or postal code’, ’State or region’, etc), rather than a textarea.
Is this already a feature?

File: app/views/application/personal-details/index.njk (hunk 1 above)
Comment (fofr) [reply app/views/application/personal-details/index.njk]: I haven't changed it here, just whitespace changes.

It's probably redundant:
```nunjucksAppEnv.addFilter('ifSetForApplication', (sections) => {
      return !!getApplicationValue(sections)
    })```

**Pull request merged**